autowarefoundation/autoware
# Support Sick LD-MRS Lidar

## Status
DEVELOPMENT

## Description
Added drivers for Sick LD-MRS lidar.
autowarefoundation/autoware_ai#155 

## Todos

## Steps to Test or Reproduce
1. Input the IP address and the frequency in config dialog next to Sick LD-MRS (IBEO Lux)
2. Launch the driver
3. Open RViz and check if the scan points (topic: `/cloud`) is published
